tests/func/test_run.py

```diff
@@ -941,3 +943,25 @@ def test_bad_stage_fname(repo_dir, dvc_repo):
 
     # Check that command hasn't been run
     assert not os.path.exists("out")
+
+
+def test_should_raise_on_stage_dependency(repo_dir, dvc_repo):
+    stage_file_name = "name.dvc"
```

Comment:
No need to have a separate var for the string that is short than the name :) Just use "name.dvc" down below.

Reply:
done